client/.gitignore

```diff
@@ -12,7 +12,7 @@ node_modules
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-pnpm-debug.log*
+npm-debug.log*
```

Comment:
Äh... warum änderst du hier pnpm auf npm?

Das npm-debug.log steht schon in Zeile 12. Pnpm ist halt ein anderer Package-Manager.

Reply:
Ich hatte erst pnpm genommen, es aber im Container nicht ans Laufen bekommen.

Dann habe ich alle pnpm per replace auf npm geändert;)